pipeline_noted.ipynb

```diff
@@ -99,14 +110,18 @@
-    "* Both `subset` and `subset_mc` are used to subset the data. `subset` operates on the whole data. A good usecase for this is to take out some troublesome frames or take out some bad pixels, for example, those used as timestamps in wireless miniscope recording. `subset_mc` operates only on the input data to motion correction, and thus only affect how the shifts are calculated. A good usecase for this is if you have good landmark (e.g blood vessels) in the field of view and want to `estimate_shifts` only on those region of interest. Note that if you are running in interactive mode (`interactive = True`), there is an interactive viewer later to help you find proper `subset_mc`. Both `subset` and `subset_mc` should be specified as python [dictionary](https://docs.python.org/3/tutorial/datastructures.html#dictionaries), whose `keys` are the dimension along which subsetting should be done, and `values` specify how subsetting should be done (usually a [slice object](https://docs.python.org/3/c-api/slice.html)). If the word \"dimension\" and how data are indexed are confusing, move on to the section where the data structure are discussed.\n",
     "* `in_memory` specify whether to load the whole dataset into memory.\n",
-    "* `interactive` controls whether interactive plots should be shown. Not that plotting the results implies actual computation to be carried out, and thus could be very inefficient when the data are not in the memory. In real situation you would probably want `in_memory = True` as well if your data can fit in the memory.\n",
-    "* `output_size` controls the realtive size of all the plots on a scale of 0-100 percent, though it can goes beyond 100 without any problem. Adjust this to please your eye."
+    "* `interactive` controls whether interactive plots should be shown. In real situation, when you want to visualize interactive plots while figuring out the best parameters, you would want `in_memory=True` and `interactive=True` as long as your data can fit in the memory. On the other hand, once you finalized your parameters and ready for batch processing, you would want both of them to be set as False.\n",
+    "* `output_size` controls the realtive size of all the plots on a scale of 0-100 percent, though it can goes beyond 100 without any problem. Adjust this to please your eye.\n",
+    "* `param_save_minian` specify the data saving property for all the saving part. `dpath` is where you want the data to be saved, we recommand use the same dpath as where you load the imaging data. `fname` is the name of your dataset. In `backend`, `'zarr'` is designed for parallel and out-of-core computation, but its support is experimental for now and does not support incremental writing (i.e. it will be a pain to update part of exsisting dataset). `meta_dict` is a `dictionary` that is used to construct meta data for the final labeled data structure. Currently we assume data are stored in some structured folders as shown below:\n",
```

Comment:
the default works with this structure, doesn't mean we have to assume this specific structure